### ISSUE TITLE
don't show the layout label over fullscreen video

### ISSUE THREAD

**Issue by MoritzFago** [ianyh/Amethyst#381]

**Comment by ianyh** [comment]
What do you mean by fullscreen video? Like a youtube video or somesuch that goes into fullscreen mode?

**Comment by MoritzFago** [comment]
yes, but maybe it would be more reasonable to disable the label on all fullscreen spaces.

**Comment by ianyh** [comment]
The label actually already is disabled for fullscreen spaces. Unfortunately, fullscreen mode video like that isn't a separate space.
